SU2_CFD/src/iteration_structure.cpp

```diff
@@ -941,6 +941,328 @@ bool CFluidIteration::MonitorFixed_CL(COutput *output, CGeometry *geometry, CSol
   return fixed_cl_convergence;
 }
 
+CNEMOIteration::CNEMOIteration(CConfig *config) : CIteration(config) { }
+
+CNEMOIteration::~CNEMOIteration(void) { }
+
+void CNEMOIteration::Preprocess(COutput *output,
```

Comment:
Is a NEMO iteration really necessary? It looks suspiciously similar to the fluid iteration.